docs/source/share_dataset.rst

```diff
@@ -103,7 +103,7 @@ Push the changes to your account using:
 Sharing a "community provided" dataset
 -----------------------------------------
 
-In this page, we will show you how to share a dataset with the community on the `datasets hub <https://huggingface.co/datasets>`__.
+In this page, we will show you how to share a dataset with the community on the `Datasets Hub <https://huggingface.co/datasets>`__.

```

Comment:
Need 🤗 here in all the instances of Datasets Hub below?

Reply:
Just added them, thanks !